src/python/pants_release/common.py

```diff
@@ -31,3 +32,17 @@ def elapsed_time() -> tuple[int, int]:
     now = time.time()
     elapsed_seconds = int(now - _SCRIPT_START_TIME)
     return elapsed_seconds // 60, elapsed_seconds % 60
+
+
+def sorted_contributors(git_range: str) -> list[str]:
```

Comment:
Moved from contributors.py and arg renamed from range to git_range, since range is a Python builtin